README.md

```diff
@@ -49,6 +49,12 @@ However, its primary use case is to notify players about the status of the tourn
 There are plans for many more services in the future.
 To learn more, see our [future plans](##Current-and-Future-State).
 
+## Contributing
+To contribute to the project, create branches off the repository itself (don't fork into your own namespace) and open PR's against the main branch. All PR's are subject to review. To run the project in development, execute the following commands:
+
+cargo install cargo-shuttle
+cargo install trunk // make sure to run these to installs separately
+rustup target add wasm32-unknown-unknown
```

Comment:
I think there are two things that are potentially missing from this list: Docker and MongoDB.

Shuttle uses Docker to construct environments to run things even locally (or at least that did a few months ago). When setting up your dev setup, did you already have Docker installed? If so, we should add that to this list.

Because I believe that Shuttle uses Docker even locally, you might not need MongoDB installed. However, because of CORS and TLS reasons, we might need to add support for devs to run Squire outside of the Shuttle runtime. In those cases, devs will need MongoDB. I assume that you already had that installed too.

Reply:
Yes, I already had those installed, which is likely why I didn't need to install them. I'm adding these changes now and will push an updated version in a few moments.